models/sipnet/R/split_inputs.SIPNET.R

```diff
@@ -12,7 +12,7 @@
 ##' @description Splits climate met for SIPNET
 ##' 
 ##' @return file split up climate file
-##'
+##' @importFrom rlang .data
```

Comment:
This line is the right one to add here 👍 

But because of it the CI checks will assign you some chores:
* edit DESCRIPTION to add rlang to the Imports section
* Rebuild the Roxygen docs (e.g. `make document`) so that it updates NAMESPACE for you
* run `./scripts/generate_dependencies.R` to update docker/depends/pecan_package_dependencies.csv